project/build.properties

```diff
@@ -13,4 +13,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-sbt.version=1.9.7
+sbt.version=1.10.1
```

Comment:
1.10.2 is out now

Reply:
sbt 1.10.2 upgrade breaks the build and needs to be worked around by upgrading sbt-pekko-build - I can do this in another PR